src/main/java/org/sigmah/client/ui/presenter/project/ProjectCalendarPresenter.java

```diff
@@ -109,14 +111,32 @@ public void onBind() {
 	 * {@inheritDoc}
 	 */
 	@Override
-	public void onPageRequest(final PageRequest request) {
-		final EnumMap<CalendarType, Integer> calendars = new EnumMap<CalendarType, Integer>(CalendarType.class);
-		calendars.put(CalendarType.Activity, getProject().getId());
-		calendars.put(CalendarType.Personal, getProject().getCalendarId());
-		calendars.put(CalendarType.MonitoredPoint, getProject().getPointsList().getId());
-		calendars.put(CalendarType.Reminder, getProject().getRemindersList().getId());
-		calendarPresenter.reload(calendars);
-	}
+	   public void onPageRequest(final PageRequest request) {
+        final Map<Integer, Integer> calendars = new HashMap<Integer, Integer>();
+        Integer CalendarId = 1;
```

Comment:
The [Java naming convention](http://www.oracle.com/technetwork/java/codeconventions-135099.html) encourage naming variables with a first letter as lowercase .
`CalendarId` should be spelled `calendarId`.